documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -834,6 +831,103 @@ public interface IJoystickInputHandler : IInputHandler
 
 `HandleHatMove` must be called when any value of `JoystickState.Hats` changes.
 
+# Input Maps
+
+## Presentation
+Input maps are designed to be easyer to the user to handle actions with more than one input.
+At first view it can looks useless and a bad practice, but this is complemtelly useful for cross-plataform programs, being games an example.
+
+This is an pseudocode example of what the Input Map system solves:
+
+Without inputMaps:
+```cs
+inputContext.Update();
+
+private Dictionary<string, bool> pressedKeys = new();
+
+inputContext.Gamepads.ThumbstickMove += @event => {Move(@event.Value)};
+inputContext.Joysticks.AxisMove += @event => {Move(@event.Value)};
+
+private Move(Vector2D<float> value)
+{
+    // Code for movement here
+}
+```
+
+With inputMaps:
+```cs
+inputContext.Update();
+
+// This single line get rid of the two in the example above. TWO ISN'T THE LIMIT!
+inputContext.Map.GetAction("Move").Axis += @event => {Move(@event.Value)};
+
+private Move(Vector2D<float> value)
+{
+    // Code for movement here
+}
+```
+
+## Implementation
```

Comment:
Consider changing this header, implementation details should be INFORMATIVE TEXT only rather than a normative part of the proposal. I know that's probably not what you intend this section to specify, which is why I say change the header.

Reply:
yeah, the section don't teach you to implement nothing, just maps the base of the system... i will fix